src/Microsoft.AspNet.TestHost/TestServer.cs

```diff
@@ -11,6 +11,7 @@
 using Microsoft.AspNet.Http.Features;
 using Microsoft.Extensions.Configuration;
 using Microsoft.Extensions.DependencyInjection;
+using Context = Microsoft.AspNet.Hosting.Internal.HostingApplication.Context;
```

Comment:
Need to revisit the design of the clients (`ClientHandler` and `WebSocketClient`) to see why we need access to the internal fields of `Context`